lib/cli/interactive-setup/dashboard-login.js

```diff
@@ -38,6 +38,11 @@ module.exports = {
       return false;
     }
 
+    if (!isDashboard) {
```

Comment:
I would remove above `isConsole` check, and simply replace it with `!isDashboard` (and keep same inapplicablity reason)

Reply:
Just to clarify, do you mean keep the same inapplicability reason as in the `MISSING_DASHBOARD_CONTEXT` or set it to the old one `CONSOLE_CONTEXT`? I am assuming the former but just wanted to check 🙏

Reply:
The old one: `CONSOLE_CONTEXT`, as this step is actually invalided if there's intentional _console_ onboarding, and not lack of intentional _dashboard_ onboarding